frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -288,10 +293,10 @@ const Profile: NextPage = () => {
             {typeof profile.subdomain === "string" ? (
               <CheckBadgeIcon alt="" />
             ) : (
-              <PencilIcon alt="" className={styles["pencil-icon"]} />
+              <LockIcon alt="" className={styles["lock-icon"]} />
```

Comment:
@Vinnl I made it so that it shows the badge when the user has subdomain and premium, pencil icon if no subdomain but has premium and lock icon by default.

Reply:
Looking good, thanks!